metabase/metabase
# fix missing horizontal scroll on pivot tables

Closes https://github.com/metabase/metabase/issues/24678

### Description

Fixes missing horizontal scroll on pivot tables when the left sidebar is too wide.

### How to verify

1) Create a pivot table with 3-4 rows so that the rows table sidebar is wide enough
<img width="339" alt="Screenshot 2024-08-01 at 9 40 29 PM" src="https://github.com/user-attachments/assets/50edb678-bb48-4963-a8f8-8ea1db42ed5d">

2) Add the pivot table on a dashboard and make the dashcard narrow so that it can only show the left sidebar which is fixed
3) Ensure the table has a horizontal scrollbar

### Demo

[Slack thread with a loom](https://metaboat.slack.com/archives/C064QMXEV9N/p1722559346051959)

### Checklist
